internal/git/git.go

```diff
@@ -21,9 +21,10 @@ import (
 )
 
 type Client struct {
-	workspace string
-	GitBranch string
-	GitSha    string
+	workspace    string
+	GitBranch    string
+	GitSha       string
+	GitTimestamp string
```

Comment:
Not sure if we should just leave this as `time.Time` instead of returning a string from the helper function.

Reply:
i'm all for keeping it time.Time instead of casting to string

Reply:
updated!